.github/workflows/deploy.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - development
```

Comment:
On review, we don't have to deploy on push to dev. Nothing is different between dev and base, and it's only getting deployed once.

I'd remove this line.